tests/proverif-noise/src/noise_kkpsk0.rs

```diff
@@ -0,0 +1,219 @@
+// Import hacspec and all needed definitions.
+use hacspec_lib::*;
```

Comment:
This shoudl be rewritten without the lib.
Let me know if you need help with that. But this only introduces dependencies that you don't want, and requires you to define those functions, that we don't want.

Reply:
These are two different things.
`hacspec_lib` is replaced by Rust core. So this is something that will never change (or only along Rust changes) and everyone will use. Here you don't need a generic interface.
The `hacspec_crypto_something` crates are dependencies that this crate is using. They can be replaced with any other implementation of the same algorithm (libcrux, or anything else). So for these it makes sense to have an abstraction.

Reply:
I've created #453 to address the abstraction layer in a follow-up.